pkg/redshift/driver/rows_test.go

```diff
@@ -126,13 +126,14 @@ func Test_convertRow(t *testing.T) {
 		{
 			name: "numeric type float4",
 			metadata: &redshiftdataapiservice.ColumnMetadata{
+				Name:     aws.String("other"),
```

Comment:
Just wondering out loud: Regarding all the special checks for `time` column (https://github.com/grafana/redshift-datasource/issues/62), I wonder why we don't ever do it for other "number" types. Or why we don't somehow ensure it is PostgresSQL before just going for the assumption it is Unix time.